src/Database/Adapter/MySQL.php

```diff
@@ -79,10 +82,11 @@ protected function getSQLIndex(string $collection, string $id, string $type, arr
      * @param string $operator
      * @param string $placeholder
      * @param mixed $value
+     * @param string $table
      * 
      * @return string
      */
-    protected function getSQLPermissions(array $roles): string
+    protected function getSQLPermissions(array $roles, $table = "table_main"): string
```

Comment:
`$table` param is unused in this method.
```suggestion
    protected function getSQLPermissions(array $roles): string
```

Reply:
Yes but used in Mariadb ,ad not having it here produces error as the method is override